linode/manager
# feat: Statically Cache Marketplace Apps

## Description 📝
- Statically cache Marketplace Apps and use them as the `initialData` for React Query

## Major Changes 🔄
- Use wrapper instead of manual fetch for class component
- Add caching

## Preview 📷

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/115251059/ef5e7cba-6f10-4b36-8594-ae0ad08b7662" /> | <video src="https://github.com/linode/manager/assets/115251059/c41b0640-c899-4a99-95ac-f47380f88729" /> |

## How to test 🧪
- Check performance and functionality of `http://localhost:3000/linodes/create?type=One-Click`
